iothub_client/samples/iothub_client_sample_upload_to_blob/windows/iothub_client_sample_upload_to_blob.vcxproj

```diff
@@ -162,21 +163,5 @@
     <ClCompile Include="..\..\..\..\deps\parson\parson.c" />
```

Comment:
><ClCompile Include="..\..\..\..\deps\parson\parson.c" /> [](start = 4, length = 56)

Using the parson lib would be good here.

Reply:
These samples don't require it. Removed these parson references. 

---
In reply to: [233574461](https://github.com/Azure/azure-iot-sdk-c/pull/708#discussion_r233574461) [](ancestors = 233574461)